infra/base-images/base-builder/Dockerfile

```diff
@@ -16,7 +16,7 @@
 
 FROM gcr.io/oss-fuzz-base/base-clang
 MAINTAINER mike.aizatsky@gmail.com
-RUN apt-get install -y git subversion jq python3 zip make libunwind8-dev binutils-dev libblocksruntime-dev
+RUN apt-get install -y curl git subversion jq python2.7 python3 zip make libunwind8-dev binutils-dev libblocksruntime-dev
```

Comment:
just making sure: these are needed for gcloud right?

Reply:
yes :(